client/src/components/genre/PodcastGenreShow.js

```diff
@@ -1,19 +1,18 @@
-import React, { useState, useEffect } from 'react'
-import PodcastTile from '../podcasts/PodcastTile.js' 
-
-const PodcastGenreShow = props => {
+import React, { useState, useEffect } from "react"
+import PodcastTile from "../podcasts/PodcastTile.js"
+import PodcastForm from "../podcasts/PodcastForm.js"
+const PodcastGenreShow = (props) => {
```

Comment:
I know this was already merged, but just calling this component `GenreShow` might be less confusing. up to you!